Forward-port of xtensa support patch to util-linux-2.22.2 by ??

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura util-linux-2.22.2/fdisks/fdiskbsdlabel.h util-linux-2.22.2.xtensa/fdisks/fdiskbsdlabel.h
--- util-linux-2.22.2/fdisks/fdiskbsdlabel.h	2012-12-12 17:04:47.898355058 -0300
+++ util-linux-2.22.2.xtensa/fdisks/fdiskbsdlabel.h	2013-03-07 18:17:32.292723061 -0300
@@ -48,7 +48,7 @@

 #if defined (__i386__) || defined (__sparc__) || defined (__arm__) || \
     defined (__mips__) || defined (__s390__) || defined (__sh__) || \
-    defined (__aarch64__) || \
+    defined (__aarch64__) || defined (__xtensa__) || \
     defined(__x86_64__) || defined (__avr32__) || defined(__cris__)
 #define BSD_LABELSECTOR   1
 #define BSD_LABELOFFSET   0
